flake.nix

```diff
@@ -82,11 +82,13 @@
                 docker
                 pip
                 ipdb
+                click
                 pyright
                 pytest
                 pytest-asyncio
                 ruff
                 syrupy
+                server
```

Comment:
I don't think we want this. For the impure shell the server should come from the environment (also see the `pip install` in `postVenvCreation`.